sidebars.js

```diff
@@ -1,468 +1,120 @@
 /**
- * Creating a sidebar enables you to:
- - create an ordered group of docs
- - render a sidebar for each doc of that group
- - provide next/previous navigation
-
- The sidebars can be generated from the filesystem, or explicitly defined here.
-
- Create as many sidebars as you want.
+ * Revised sidebar configuration for the MultiversX documentation,
+ * designed to help new developers navigate the learning resources effectively.
  */
 
 // @ts-check
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  // tutorialSidebar: [{ type: "autogenerated", dirName: "." }],
-
-  // But you can create a sidebar manually
-  /*
   tutorialSidebar: [
-    'intro',
-    'hello',
     {
       type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
+      label: 'Getting Started',
+      items: [
+        'welcome/welcome-to-multiversx',
+        'technology/architecture-overview',
+        'technology/glossary',
+        'technology/entities',
+        'technology/chronology',
+        'setup/installing-mxpy',
+        'setup/configuring-mxpy',
```

Comment:
- people should not be installing mxpy (yet) in "getting started"
 - `configuring-mxpy` is low-level detail, this isn't about "getting started".
 - is `cli-overview` an existing page? (oh, now I see, new page, but about what?)
 - is `setup/configuration` an existing page? (oh, now I see, new page, but about what?)

I'd vote to exclude these from getting started:
 - 'setup/installing-mxpy',
 - 'setup/configuring-mxpy',
 - 'setup/cli-overview',
 - 'setup/build-reference',
 - 'setup/configuration',

Instead, maybe:
 - wallet(s) - what are they & how-to
 - explorer how-to